ConfigFiles/PlusDeviceSet_Server_StealthLinkTracker.xml

```diff
@@ -13,6 +13,7 @@ Tools: Stylus (Passive Planar, Blunt) , Reference (Small Passive Cranial Frame)"
       RegistrationUpdatePeriod="0.0"
       ToolReferenceFrame="Tracker" >
       <DataSources>
+        <!-- Remove tools that are not defined by the current StealthLink server -->

```

Comment:
Add note here to remind users not to forget removing corresponding lines from OutputChannels section.
Instead of "StealthLink server" maybe use "StealthStation" instead.